plugins/destination/sqlite/client/migrate.go

```diff
@@ -132,51 +130,53 @@ func (*Client) canAutoMigrate(changes []schema.TableColumnChange) bool {
 
 // This is the responsibility of the CLI of the client to lock before running migration
 func (c *Client) MigrateTables(ctx context.Context, msgs message.WriteMigrateTables) error {
+	allTables := schema.Tables{}
```

Comment:
This seems like a lot of changes but it only changes the logic that we first initialize all tables from all messages instead of going over each message one by one